CDCgov/prime-reportstream
# Specimen Id for ORC-3-1 and ORC-2-1

This PR makes changes to the imagemover schema so that ORC-3-1 and ORC-2-1 are populated with the specimen id. Initially, I had created  a property (useSpecimenIdForFillerOrderNumber) that would make these changes. The default covid-19 schema maps these fields to the specimen id already but it wouldn't map them for test results coming from imagemover. This is why I opted to edit the imagemover schema instead of adding the property to the translation configuration file.

Test Steps:
1. Create fake data csv file with imagemover as sender and MA for state
2. Submit curl to the api
3. Check HL7 message and verify that ORC-3-1 and ORC-2-1 matches the speciment id on MSH-10 or SPM-2-1
4. Repeat for NJ
5. Run gradlew test, testIntegration, testSmoke

## Changes
- maps placer_order_id and filler_order_id to specimen id in the imagemover schema

## Checklist

### Testing


## Fixes
- #2761  
- #2634 

## Unblocks
#2348 
#2636 
#2641
